crates/typst/src/layout/flow.rs

```diff
@@ -856,6 +960,44 @@ impl<'a, 'e> FlowLayouter<'a, 'e> {
 
         Ok(())
     }
+
+    fn layout_line_number(&mut self, numbering: &Numbering, y: Abs) -> SourceResult<()> {
+        let line_counter = Counter::of(ParLineMarker::elem());
```

Comment:
Why do we use the `ParLineMarker` counter instead of `ParLine`?

Reply:
This was actually an accident (a "bug"), which has now been promoted into a feature (:sunglasses:), as we don't want to expose the line number counter for now, until we find a good way to allow consistent counter value fetching and updating.